### DISCUSSION TITLE
How do I have the bot send a direct message?

### DISCUSSION THREAD

**Discussion by Mattw11486** [microsoft/BotFramework-Composer#5617] [Q&A]
I have the composer working in both slack/teams and able to use it through the bot framework emulator.

Everything is working fine I wish there was some more documentation on how to actually customize the templates.
For example, I am not really finding much information on how to update an adaptive card to be more of a message block that contains titles/links to other websites. Or is there a better approach to that? 

Also, how do I send a direct message to a user if they ask a question in a public channel my bot is in? Or even better, have the bot message them directly with a welcome message once they join the slack/teams channel.

**Comment by a-b-r-o-w-n** [comment]
In regards to  adaptive cards, the documentation can be found [here](https://docs.microsoft.com/en-us/adaptive-cards/). To update a card, you can use the `Update activity` action.

![image](https://user-images.githubusercontent.com/3619060/106020716-e9d00c00-6078-11eb-9693-92d3bc6f4856.png)

I will have to ask a colleague about how to send direct  messages, but I think it will depend on the channel and may not be supported yet in Composer -- but I'll get back to you.

**Reply by Clooney24** [reply]
Hi, I'm also using Composer and am in need to send direct messages days after the initial dialog (which will be triggered by our main web application).
